tests/test_hardware_api_trading_bot.py

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
As a general comment, please use `set_pv_power_forecast ` and `set_load_power_forecast ` under the method `on_market_cycle `as an example for user to send its agent's forecasted energy consumption/production for the next market.

Reply:
@faizan2590 I added a separate Script for that: https://github.com/gridsingularity/d3a-api-client/pull/59/files#diff-9e7b1acf31c4255a76caade7f40642b13f1b09f02d5dae95b0a598cb75383e51
(to simulate that the trading bot will not send the forecast in real live)